glustercli/cmd/volume-profile.go

```diff
@@ -0,0 +1,125 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/gluster/glusterd2/pkg/api"
+
+	"github.com/olekukonko/tablewriter"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Profile Info Flags
+	flagProfileInfoPeek            bool
+	flagProfileInfoIncremental     bool
+	flagProfileInfoIncrementalPeek bool
+	flagProfileInfoCumulative      bool
+	flagProfileInfoClear           bool
+)
+
+var volumeProfileCmd = &cobra.Command{
+	Use:   "profile",
+	Short: "Gluster volume profile",
+	Long:  "Gluster Volume Profile retrieves info on stats like latency, no of fops performed on the volume etc",
+	Args:  cobra.ExactArgs(2),
```

Comment:
@vpandey-RH have a question here when `Run` is implemented for this command?
what are the required 2 `Arguments` for this command?

Reply:
This is just an upper layer to " info " command. We don't need anything executing at this level. We want the CLI to execute when we give info + volname as arguments. 
The 2 arguments required at this level are "info" and <volname>. I did this just in case we need a start and stop command as well instead of using volume-set, in future. Then it can be extended to include "start"/"stop" and <volname> as its two arguments.